timber/build.gradle

```diff
@@ -35,8 +35,8 @@ dependencies {
   testCompile deps.festassert
   testCompile deps.junit
   testCompile deps.robolectric
-
   lintChecks project(path: ':timber-lint', configuration: 'lintChecks')
+  compile files('lib/jetbrains-annotations.jar')
```

Comment:
Not one that I could find AND I forgot to note, you should probably check the license on that.  It has to be extracted from inside IntelliJ/Android Studio.  Maybe  you could talk them in to uploading it to MavenCentral?